test/variable.jl

```diff
@@ -65,6 +65,21 @@ facts("[variable] get and set bounds") do
     @fact getUpper(fixedvar) --> 5
 end
 
+facts("[variable] get and set values") do
+    m = Model()
+    @defVar(m, x[1:3])
+    x0 = [1:3]
+    setValue(x.innerArray, x0)
```

Comment:
Hmm. I see from the test logs that your Julia/JuMP version is not happy with calling .innerArray. Whilst my setup is not happy with setValue(x, x0) ...